### DISCUSSION TITLE
Add an option to re-bake LightProbes without re-baking the LightmapGI

### DISCUSSION THREAD

**Discussion by jcostello** [godotengine/godot-proposals#7189] [3D]
Baking LightmapGI is time consuming. Re-Baking light probes without baking the LightmapGI would be helpful when manually placing lightprobes in the scene It would be helpful to have button light the LightmapBake but only for probes. Is that technically posible?

**Comment by SpockBauru** [comment]
Indeed, for scenes with many moving objects to fine tuning the light probes is a necessity! Specially when there are too many "blacked out" probes.

Edit:
Also is a common practice to let less important objects to be lit by the probes in order to reduce the usage of lightmap textures. This is specially important for multimeshes with numerous instances like grass.

**Reply by Calinou** [reply]
>  Specially when there are too many "blacked out" probes.

There are several ways we could address this at an engine level, e.g. https://github.com/godotengine/godot/pull/83420.

**Edit:** Check the comment I just posted on that PR too :slightly_smiling_face:

**Comment by Calinou** [comment]
Note that https://github.com/godotengine/godot/pull/82915 caches lightmap probe generation to speed up the lightmap baking process if the probes haven't changed since the last bake.

**Reply by jcostello** [reply]
I think that is the minor of the problem. The idea es to rebake the probes without baking the lightmaps. This is usefull if you are working on the position of the probes to better dynamic lighting
